src/pages/Main/Carousel/Carousel.js

```diff
@@ -1,6 +1,72 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import S from './Styled.Carousel';
```

Comment:
스타일 파일의 리뷰에 따라 해당 import 부분도 변경되어야 합니다.
```suggestion
import * as S from './Styled.Carousel';
```
이렇게 한 번 참고해보세요!